### PR TITLE
Add html css and linters files

### PR DESCRIPTION
**Pull request by Oklukeok**
In this branch was added:
-HTML file with the basic content of the project.
-CSS file to stylish the HTML file.
-Linter files to find any possible error.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Oklukeok 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that has linter checks that do not pass. Kindly make sure that you fix all linter errors before requesting a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        feature or website Look through some of my work and experience!
+         If you like what you see and have a project you need coded, 
+         don’t hestiate to contact me.</p>
+      </div>
+      <div class="landingH3"><h3>LET’S CONNECT</h3></div>
+      <div class="socialMedia">
+        <i class="fa-brands fa-linkedin"></i>
+        <i class="fa-brands fa-square-github"></i>
+        <i class="fa-brands fa-square-twitter"></i>
+      </div>
```
Comment (mwanawabangona) [inline index.html:61]: - [ ] `required`
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
* Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.

something like this
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  <header>
+
+    <div class="phoneItems">
+
+      <div class="phone phoneTime"><p>9:41</p></div>
+      <div class="phone phoneSpace"></div>
+      <div class="phone phoneSignal"><img src="https://cdn-icons-png.flaticon.com/512/39/39934.png"  alt="phone signal icon"></div>
+      <div class="phone phoneWifi"><img src="https://cdn-icons-png.flaticon.com/512/93/93158.png" alt="phone wifi icon"></div>
+      <div class="phone phoneBattery"><img src="https://cdn-icons.flaticon.com/png/512/664/premium/664886.png?token=exp=1660853218~hmac=9c898731a4907abfe6c540e736a70f45" alt="phone battery icon"></div>
+
```
Comment (mwanawabangona) [inline index.html:36]: - [ ] `optional`
- [ ] Good job all the way to follow the template how its a requirement to add the battery, network, wifi and signal.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Oklukeok ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Oklukeok  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**